server/catgenome/src/main/java/com/epam/catgenome/manager/target/TargetGeneManager.java

```diff
@@ -460,7 +475,7 @@ private static boolean isOptions(final List<String> filedValues) {
         return filedValues.stream().distinct().count() / filedValues.size() < OPTIONS_RATIO;
     }
 
-    private static void setIds(final long targetId, final List<TargetGene> targetGenes) {
+    private void setIds(final long targetId, final List<TargetGene> targetGenes) {
```

Comment:
I think that calling database to get next sequence ID for each gene maybe really slow. Can we request all ids at first and then assign them?